Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 523
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 524
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\billable_hours\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\billable_hours\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 524
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
Bad Request: /api/v1/timetable
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\billable_hours\urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\billable_hours\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\billable_hours\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 524
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 524
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 89
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 472
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 472
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 472
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 122
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 135
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 356
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 70
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 472
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 612
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 612
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\serializers.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 496
"POST /api/v1/timetable HTTP/1.1" 201 496
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 496
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 59
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 59
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 496
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 43
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 497
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 83
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 83
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 83
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 83
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 73
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 119
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 119
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 119
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 119
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\pdf_generator.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
"POST /api/v1/timetable HTTP/1.1" 201 500
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 200 2047
"POST /api/v1/timetable HTTP/1.1" 200 2047
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\billable_hours\urls.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 200 2047
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\renderers.py", line 99, in render
    ret = json.dumps(
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\utils\json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\utils\encoders.py", line 67, in default
    return super().default(obj)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type HttpResponse is not JSON serializable
"POST /api/v1/timetable HTTP/1.1" 500 104058
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 77
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\renderers.py", line 99, in render
    ret = json.dumps(
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\utils\json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\utils\encoders.py", line 67, in default
    return super().default(obj)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python310-32\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type HttpResponse is not JSON serializable
"POST /api/v1/timetable HTTP/1.1" 500 104040
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 544
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 201 500
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 200 2047
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
"POST /api/v1/timetable HTTP/1.1" 400 30
"POST /api/v1/timetable HTTP/1.1" 200 2055
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 200 2055
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 200 2055
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py", line 177, in create_timetable
    timetable_processor = TimetableProcessor(request)
TypeError: TimetableProcessor.__init__() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 93688
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py", line 177, in create_timetable
    timetable_processor = TimetableProcessor(request)
TypeError: TimetableProcessor.__init__() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 93688
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py", line 177, in create_timetable
    timetable_processor = TimetableProcessor(request)
TypeError: TimetableProcessor.__init__() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 93688
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py", line 177, in create_timetable
    timetable_processor = TimetableProcessor(request)
TypeError: TimetableProcessor.__init__() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 93688
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py", line 177, in create_timetable
    timetable_processor = TimetableProcessor(request)
TypeError: TimetableProcessor.__init__() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 93696
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
TypeError: create_timetable() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 90067
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /api/v1/timetable
Traceback (most recent call last):
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
TypeError: create_timetable() missing 1 required positional argument: 'invoice_response'
"POST /api/v1/timetable HTTP/1.1" 500 90067
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /api/v1/timetable HTTP/1.1" 200 2055
C:\Users\USER\Desktop\external projects\affinity_labs_take-home_assessment\billable_hours\timetracking\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /api/v1/timetable
Bad Request: /api/v1/timetable
Not Found: /api/v1/timetables
Bad Request: /api/v1/timetable
Bad Request: /api/v1/timetable
